model/metricset/_meta/fields.yml

```diff
@@ -78,3 +78,58 @@
             format: bytes
             description: >
               The Resident Set Size. The amount of memory the process occupied in main memory (RAM).
+
+
+- key: apm-transaction
```

Comment:
This is not nested under `transaction`, same for `span`. They are both top-level. That's the reason for [the failing test](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-server%2Fapm-server-mbp%2FPR-2315/detail/PR-2315/3/pipeline/).